_scripts/build-storybooks-indexes.js

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env node
+/* eslint no-use-before-define: 0 */
+const fs = require("fs");
+const path = require("path");
+
+function main() {
+  const [, , PUBLISH_ROOT] = process.argv;
+
+  const commitsPath = path.join(PUBLISH_ROOT, "commits");
+  const pullsPath = path.join(PUBLISH_ROOT, "pulls");
+
+  const pullDirs = lsDirs(pullsPath);
+  const commitDirs = lsDirs(commitsPath);
+
+  // Create an index for all pull requests
+  writeIndex(
+    pullsPath,
+    htmlDirIndex({
+      title: "Storybooks for Recent Pull Requests",
+      items: pullDirs
+    })
+  );
+
+  // Create an index for each pull request
+  for (const item of pullDirs) {
+    writeIndex(item.path, htmlPullIndex(item));
+  }
+
+  // Create an index for all commmits
+  writeIndex(
+    commitsPath,
+    htmlDirIndex({
+      title: "Storybooks for Recent Commits",
+      items: commitDirs
+    })
+  );
+
+  // Create an index for each commit
+  for (const item of commitDirs) {
+    writeIndex(
+      item.path,
+      htmlDirIndex({
+        omitDates: true,
+        title: `Storybooks for ${item.fn}`,
+        description: item.description,
+        items: lsDirs(item.path)
+      })
+    );
+  }
+
+  // Create an index for the whole static site
+  writeIndex(
+    PUBLISH_ROOT,
+    htmlIndex({
+      pulls: pullDirs.slice(0, 7),
+      commits: commitDirs.slice(0, 7)
```

Comment:
Yeah, the problem is: Which most recent change? I should probably add master in there as a baseline, but we're likely to have multiple PRs in flight at once. If we only hosted one recent change, that wouldn't help preview & review what's in progress before landing.

Reply:
Yeah, I figured it would just be whatever was latest from landing on master. That could be an improvement here to have builds from master at the top of the page, especially if UX wants to poke around but is confused by the interface, maybe as another follow up if we think that'd be valuable. 😀